### ISSUE TITLE
Move docs to dedicated repo

### ISSUE THREAD

**Issue by pwalsh** [okfn/opendatacensus#758]
# Description

Same pattern as OS and OT

# Tasks

**Comment by brew** [comment]
Updating documentation should be part of the process of making changes to an applications source code.

tl;dr: I propose hosting the docs within the application source code repository, in a `docs` subdirectory, and build using ReadTheDocs with corresponding versioning based on repository tags.

---

The Open Data Census documentation is currently a Jekyll based-project hosted using github Pages. The source is the `gh-pages` branch of the OpenDataCensus github repository (within the okfn organization) [1]. The master branch of the opendatacensus repo hosts the source code for census project.

Managing docs in this way presents a couple of issues:

1. the docs source is effectively hidden away in the gh-pages branch
2. updating and versioning the docs becomes a separate process from updating the census source code (it shouldn't be). Typically requiring separate, disconnected pull requests.

OpenSpending and OpenTrials host their docs on ReadTheDocs (RTD) [2], with their doc source hosted on individual github repos within their respective organizations [3]. RTD provides a familiar style and organization to documentation across projects, and helps maintain versions (including master and stable). This allows us to update our documentation on master without going out-of-sync with live documentation for the latest stable release.

However, the docs for OS and OT are maintained in separate repositories from their corresponding application source code. This loses some of the benefits of keeping source and documentation together, so they can be updated within the same feature branch and pull request.



---

[1] https://github.com/okfn/opendatacensus/tree/gh-pages
[2] http://openspending.readthedocs.io/en/latest/ & http://opentrials.readthedocs.io/en/latest/
[3] https://github.com/openspending/docs & https://github.com/opentrials/docs

**Comment by pwalsh** [comment]
I'm neutral. I think subjectively that separation is better, but totally happy for your choice.

**Comment by brew** [comment]
Fixed in #763
